src/plotman/job.py

```diff
@@ -25,12 +25,13 @@ def job_phases_for_dstdir(d, all_jobs):
     return sorted([j.progress() for j in all_jobs if j.dstdir == d])
 
 def is_plotting_cmdline(cmdline):
+    if cmdline and 'python' in cmdline[0]:
+        cmdline = cmdline[1:]
     return (
-        len(cmdline) >= 4
-        and 'python' in cmdline[0]
-        and cmdline[1].endswith('/chia')
-        and 'plots' == cmdline[2]
-        and 'create' == cmdline[3]
+        len(cmdline) >= 3
+        and cmdline[0].endswith("chia")
```

Comment:
Was the `/` removed intentionally?  If so, can you share an example command line from `psutil` where there is no slash?

```suggestion
        and cmdline[0].endswith("/chia")
```

Reply:
Yeah it was intentional - I should have mentioned that in the commit message. When running the `chia` from the PATH, it looks like this, with no leading slash:
```
% ps -axf | grep chia
  501 25158 25147   0  9:56am ttys003    0:13.87 chia plots create
```

Reply:
Alrighty, sounds good for now.  The pure string methods approach is a bit insufficient.  It should probably be a regex like `"[$/]chia^"` or something.  But, that can be addressed differently.  I want to avoid stuff like scripts named `plot_chia` being identified as chia plotting processes.  Unlikely given all the other checks, but still.  Anyways, as mentioned, improving that is a separate task I'll take on if I really want to.  `:]`